test/disability-benefits/526EZ/helpers.unit.spec.jsx

```diff
@@ -1,8 +1,12 @@
 import { expect } from 'chai';
-import { flatten } from '../../../src/js/disability-benefits/526EZ/helpers.jsx';
+import {
+  flatten,
+  isPrefillDataComplete,
+  prefillTransformer
+} from '../../../src/js/disability-benefits/526EZ/helpers.jsx';
 import initialData from './schema/initialData.js';
 
-const formData = initialData;
+delete initialData.prefilled;
```

Comment:
This may cause problems for some other tests. May not need to worry about it for now, but if we get some weird failures later on down the road, this might be why.

JS is weird. :man_shrugging: